tests/test-menubar.lua

```diff
@@ -3,6 +3,12 @@
 local runner = require("_runner")
 local menubar = require("menubar")
 
+local menubar_refreshed = false
+
+function menubar.refresh(scr)
+    menubar_refreshed = scr or true
+end
```

Comment:
How about
```lua
local orig_refresh = menubar.refresh
function menubar.refresh(...)
    menubar_refreshed = true -- What was your scr or true supposed to do?
    orig_refresh(...)
end
```
That way more of the original code runs and gets checked for "does it cause errors"?